docs/.vuepress/config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
     title: 'Splash Documentation',
     description: 'Documentation for the Splash server',
-    base: '/docs/',
+    base: '/help/',
```

Comment:
I actually think that /docs was a better word for the link. "Help" is a better word for the user in the index.